### PR TITLE
homed: allow per-user additional LUKS mount options

### PR DESCRIPTION
**Pull request by poettering**
Fixes: #15120

### CODE REVIEW

File: man/homectl.xml [hunk 1]
```diff
@@ -656,6 +656,13 @@
         to on, to ensure disk space is minimized while a user is not logged in.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--luks-extra-mount-options=</option><replaceable>OPTIONS</replaceable></term>
+
+        <listitem><para>Takes a string containing additional mount options to use when mounting the LUKS
+        volume. If specified, this string will appended to the default mount options.</para></listitem>
```
Comment (bluca) [inline man/homectl.xml]: "will be"

**Review by bluca — APPROVED** [review]
just one typo in the docs

**Comment by poettering** [comment]
Force pushed new version with the typo fixed. Upgrading green label

**Review by yuwata — APPROVED** [review]
LGTM.

**Pull request merged**